thirdeye/thirdeye-frontend/app/pods/components/rootcause-header/component.js

```diff
@@ -24,6 +24,12 @@ export default Component.extend({
    */
   sessionText: null,
 
+  /**
+   * Rootcause session owner
+   * @Type {string}
```

Comment:
nit: `@type` doesn't need to be capitalized.

Reply:
fixed